src/components/templates/PartyMap/components/Map/Map.tsx

```diff
@@ -160,7 +160,7 @@ export const Map: React.FC<PropsType> = ({
 
   const { partygoersBySeat, isSeatTaken } = usePartygoersbySeat({
     venueId,
-    partygoers,
+    partygoers: recentWorldUsers,
   });
```

Comment:
This is going to change the logic, as previously, `usePartygoers` would only include people in this venue, whereas `worldUsers` expands that concept.

Essentially, I believe every place where you've changed `usePartygoers` to `useRecentWorldUsers` is probably going to be wrong.